### DISCUSSION TITLE
Spanish Dictionary (wordlist)

### DISCUSSION THREAD

**Discussion by alejandracios** [keepassxreboot/keepassxc#9854] [Ideas]
Hi, 
just want to share a Spanish wordlist for password generator. I'm spanish speaker and couldn't find any good worldlist. Not saying mine is good but have done some work I couldn't find on others. 
My source is Real Academia Española Corpus de Referencia del Español Actua (RAE, CREA). Took the file and remove all the words with little frequence and words with less than 2 letters (with the exception of common use articles and pronouns). This gave me a list of 136K words. Probably requires more refinement. 

Addtionally I'm sharing a second file without tilde (accent mark = á, é, í, ó, ú) in case password doesn't accept special characters  and a third file without ñ (in case that letter is taken as special character too). Those files contains 132K words as when removing that "special characters" I removed the dupicated words. 

Hope you can use a good use of it. 

[Diccionario Español 136k palabras.txt](https://github.com/keepassxreboot/keepassxc/files/12651434/Diccionario.Espanol.136k.palabras.txt)

[Diccionario Español no special characters (no tilde).txt](https://github.com/keepassxreboot/keepassxc/files/12651435/Diccionario.Espanol.no.special.characters.no.tilde.txt)
[Diccionario Español no special charanters (no tilde no ñ).txt](https://github.com/keepassxreboot/keepassxc/files/12651436/Diccionario.Espanol.no.special.charanters.no.tilde.no.n.txt)

**Comment by philgib1** [comment]
Very nice but you should encode it in unicode UTF-8    ;-)
